books/models.py

```diff
@@ -0,0 +1,59 @@
+from django.db import models
+
+from utils        import timer
```

Comment:
Class명은 대문자로 시작해야 합니다. timer -> Timer

```suggestion
from utils        import timer
```

Reply:
수정하였습니다